src/main/java/org/opensearch/knn/index/codec/util/KNNCodecUtil.java

```diff
@@ -115,7 +115,7 @@ public static String buildEngineFileSuffix(String fieldName, String extension) {
         return String.format("_%s%s", fieldName, extension);
     }
 
-    private static long getTotalLiveDocsCount(final BinaryDocValues binaryDocValues) {
+    public static long getTotalLiveDocsCount(final BinaryDocValues binaryDocValues) {
```

Comment:
This KNNCodecUtil class will be refactored, once I integrate the NativeIndexWriter functionality with KNNVectorsFormatCodec.